mesonbuild/mformat.py

```diff
@@ -925,7 +926,13 @@ def add_arguments(parser: argparse.ArgumentParser) -> None:
     inplace_group.add_argument(
         '-q', '--check-only',
         action='store_true',
-        help='exit with 1 if files would be modified by meson format'
+        help='silently exit with 1 if files would be modified by meson format'
+    )
+    parser.add_argument(
```

Comment:
I think this should be part of the mutually exclusive group as well

Reply:
Ah good catch, sorry just copy-pasted it